Support/bone101/index.html

```diff
@@ -170,7 +170,7 @@ <h2 id="node-red"><a target="_blank" class="node-red-url" href="http://192.168.7
  <h2 id="cloud9"><a target="_blank" class="cloud9-url" href="http://192.168.7.2:3000">Cloud9 IDE</a></h2>
  <p>To begin editing programs that live on your board, you can use the Cloud9 IDE.</p>
  <p>Click on the "Cloud9 IDE" link above to start the editor.</p>
- <a target="_blank" class="cloud9-url">
+ <a target="_blank" class="cloud9-url" href="http://192.168.7.2:3000">
```

Comment:
We'll need to look at the replacement script for this URL such that this is the IP address of the board where we are connected.